src/app/frontend/variables.scss

```diff
@@ -20,6 +19,8 @@ $hover-primary: #1254df;
 $body: #eee;
 $emphasis: #000;
 $content-background: #fff;
-// TODO(bryk): Reference angular_material/src/core/style/variables.scss instead of having
-// this variable here.
-$whiteframe-shadow-z1: 0 2px 5px 0 rgba(0, 0, 0, .26) !default;
+
+// TODO(bryk): Get those variables from Angular Material scss files.
+$layout-breakpoint-lg: 1280px;
+$whiteframe-shadow-1dp: 0 1px 3px 0 rgba(0, 0, 0, .2), 0 1px 1px 0 rgba(0, 0, 0, .14), 0 2px 1px -1px rgba(0, 0, 0, .12);
+$baseline-grid: 8px;
```

Comment:
I'm also not a big fan of this name, but this is what Angular Material uses. Take a look at: https://github.com/angular/material/blob/HEAD/src/core/style/variables.scss

I hope that soon we'll be able to include angular material scss, so then this variable here will be deleted. And Material's one used. What do you think? Can I leave the name as it is?